### PR TITLE
feat/sent requests

### PR DESCRIPTION
**Pull request by ghost**
added the SentRequests Component to view sent requests by the user (hardcoded for now)

### CODE REVIEW

**Comment by ghost** [comment]
the number of commits on this are because I initially branched off of `feat/view-requests`. It is up to date with dev

**Comment by davidkliewer-kirby** [comment]
> the number of commits on this are because I initially branched off of `feat/view-requests`. It is up to date with dev

Nice compact PR though!

**Comment by ghost** [comment]
> > the number of commits on this are because I initially branched off of `feat/view-requests`. It is up to date with dev
> 
> Nice compact PR though!

thought I'd make this one a little smaller considering the last one as 1200 lines 😆

File: src/Components/Scenes/ViewRequests/SentRequests/SentRequests.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import RequestTableTitle from '../RequestTableTitle';
+import RequestTable from '../RequestTable';
+import { transformRequests } from '../../../../State/helpers';
+
+// Approver email is hard-coded until authentication is implemented
+const userEmail = 'jonathan.delarosa@pnmac.com';
+const requestsInboxTableColumns = [
```
Comment (lheld-pnmac) [inline src/Components/Scenes/ViewRequests/SentRequests/SentRequests.js]: Small naming nit - should this be `sentRequestsTableColumns`?

File: src/Components/Scenes/ViewRequests/SentRequests/SentRequests.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  { name: 'Request', property: 'databasename' },
+  { name: 'Description', property: 'description' },
+  { name: 'Status', property: 'requeststatus' },
+  { name: 'Date Requested', property: 'createddate' }
+];
+
+const SentRequests = props => {
+  const { requests, userRequestsFetch } = props;
+
+  // Fetch all inbound requests given for approver's email
```
Comment (lheld-pnmac) [inline src/Components/Scenes/ViewRequests/SentRequests/SentRequests.js]: Nit - update this comment for SentRequests

**Review by lheld-pnmac — COMMENTED** [review]
Just a variable name and comment I noticed could be updated. Everything else looks great!

**Review by Jake-Lindvall — COMMENTED** [review]
Agree with the naming/commenting things for clarity that Libbey said. Otherwise, looks great, and thanks for the small and digestible MR :)

**Comment by ghost** [comment]
> Just a variable name and comment I noticed could be updated. Everything else looks great!

Good catches! have been resolved :)

**Review by lheld-pnmac — APPROVED** [review]
Looks good!

**Review by davidkliewer-kirby — APPROVED** [review]

**Pull request merged**